### PR TITLE
fix(#11): add support for older node versions

### PR DESCRIPTION
**Pull request by oxmanroman**
**Description**
Add support for older Node versions by using `fs-extra`

### CODE REVIEW

File: src/generate.ts [hunk 1]
```diff
@@ -55,16 +56,18 @@ const outputPath = path.resolve(argv.output);
 export async function run(): Promise<void> {
   const templateUrl = (templates as { [name: string]: string })[argv.template];
 
-  // clone selected template
-  console.info(`Setting up template: ${argv.template}`);
-  if (fs.existsSync(templatePath)) {
+  // download selected template
+  if (fs.pathExistsSync(templatePath)) {
```
Comment (chekuda) [inline src/generate.ts:60]: so this is returning a falsy value?

File: src/generate.ts [hunk 2]
```diff
@@ -55,16 +56,18 @@ const outputPath = path.resolve(argv.output);
...
   } else {
-    shell.exec(`git clone ${templateUrl} ${templatePath}`);
+    console.info(`Downloading template: ${argv.template}`);
+    await downloadRepository(templateUrl, templatePath);
+    console.info(`Template cached at: ${templatePath}`);
   }
   shell.cd(templatePath);
 
   // install template dependencies
+  console.info(`Installing template dependencies, go grab some coffee`);
```
Comment (chekuda) [inline src/generate.ts:70]: Love the message!!

File: src/templates.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "navy": "https://github.com/lightning-resume/template-navy-react.git"
+  "navy": "https://github.com/lightning-resume/template-navy-react/archive/main.zip"
```
Comment (chekuda) [inline src/templates.json:2]: why need this??

**Review by chekuda — APPROVED** [review]

File: src/templates.json (hunk 3 above)
Comment (oxmanroman) [reply src/templates.json:2]: In order to download the zip directly and avoid cloning the repo and relying on the user's git setup

File: src/generate.ts (hunk 2 above)
Comment (oxmanroman) [reply src/generate.ts:70]: :heart:

File: src/generate.ts (hunk 1 above)
Comment (oxmanroman) [reply src/generate.ts:60]: It's actually a boolean, it does the same as `existsSync`, I just wanted to use the `fs-extra` syntax

**Pull request merged**